Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -554,21 +833,40 @@ - (void)prepareForReuse
 //{
 //    return [UIColor whiteColor];
 //}
+
+#pragma mark - Notifications
+
+- (void)setIsCellVisible:(BOOL)isCellVisible {
+    if (self.isCellVisible == isCellVisible) {
+        return;
+    }
+    
+    [super setIsCellVisible:isCellVisible];
```

Comment:
Might we want to call super regardless of if `self.isCellVisible == isCellVisible`?

Reply:
Done.